### PR TITLE
Added custom input to the quotes

### PR DESCRIPTION
**Pull request by Omnigazer**

### CODE REVIEW

File: starWarsQuotesRepository.py [hunk 1]
```diff
@@ -39,6 +40,15 @@ def __getQuotes(self, trilogy: str = None) -> List[str]:
 
         return result
 
+    def __processQuote(self, quote: str, input: str = None) -> str:
+        reg = re.compile('\$\((.*)\)')
```
Comment (charlesmadere) [inline starWarsQuotesRepository.py]: This looks awesome, but I want you to make a tiny change here 😉 

Can you move this to the constructor, like I have [here in `TriviaGameRepository`](https://github.com/charlesmadere/CynanBotCommon/blob/develop/triviaGameRepository.py#L102)? So then we would only compile this regex when the `StarWarsQuotesRepository` is first instantiated. Maybe name it `self.__quoteRegEx`?

File: starWarsQuotesRepository.py [hunk 2]
```diff
@@ -39,6 +40,15 @@ def __getQuotes(self, trilogy: str = None) -> List[str]:
 
         return result
 
+    def __processQuote(self, quote: str, input: str = None) -> str:
```
Comment (charlesmadere) [inline starWarsQuotesRepository.py:44]: Can you add a quick return here? Like this:

```
if not utils.isValidStr(input):
    return quote
```

**Review by charlesmadere — COMMENTED** [review]
pls change just 2 tiiiiinnyyyyy things 🤓 👍🏻

**Review by charlesmadere — APPROVED** [review]

File: starWarsQuotesRepository.py (hunk 1 above)
Comment (Omnigazer) [reply starWarsQuotesRepository.py]: You're right, that makes sense. Done :)

File: starWarsQuotesRepository.py (hunk 2 above)
Comment (Omnigazer) [reply starWarsQuotesRepository.py:44]: Done.

**Pull request merged**